exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,15 @@
 // Start by creating a variable `message`
+var message = "My name is ";
+var name = "Shafiek";
+var sentence = " and my name is ";
+var nameLength = name.length;
+var sentenceEnd = " characters long";
+var fullSentence = message + name + sentence + nameLength + sentenceEnd;
 
-console.log(message);
+console.log(fullSentence);
```

Comment:
Nice work Shafiek 🔥 
Works as expected, but maybe not all the parts of the string needs to be seperate variables.
Using [template literals ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) will make the code less cluttered and achieve the same result.